gateway/src/apicast/policy/3scale_batcher/README.md

```diff
@@ -0,0 +1,62 @@
+# 3scale Batcher Policy
+
+## Description
+
+The APIcast policy performs one call to the 3scale backend for each request that
+it receives. The goal of this policy is to reduce latency and increase
+throughput by significantly reducing the number of requests made to the 3scale
+backend. In order to achieve that, this policy caches authorization statuses and
+batches reports.
+
+## Technical details
+
+When the APIcast policy receives a request, it makes an 'authrep' call to
+backend. This call checks the credentials sent by APIcast, and also applies rate
+limiting over the metrics sent by APIcast. If the credentials are correct and
+rate limits not violated, backend also increases the counters of the metrics
+reported by APIcast. This counters are used both to show statistics in the
+3scale UI and also to apply the rate limits. This means that the rate limiting
+applied will not be accurate until the counter is updated. For limits defined
+for long windows of time (hour, day, etc.) this update lag if often irrelevant.
+However, it might be important to take it into account for limits defined for a
+small window of time (a per-minute limit, for example).
+
+This policy uses a cache for authorizations and batches reports. Also, it makes
+'authorize' and 'report' calls to backend instead of 'authrep' calls. On each
+request, the flow is as follows: 1. The policy checks whether the credentials
```

Comment:
Might be good to make the numbered list actual numbered list so it is rendered as line items.

Reply:
Done :+1: